lib/rules/sotd/draft-stability.js

```diff
@@ -8,7 +8,11 @@ const self = {
 exports.name = self.name;
 
 exports.check = function (sr, done) {
-    if (!sr.config.stabilityWarning || sr.config.stabilityWarning === 'REC')
+    if (
```

Comment:
I don't know why this hasn't been removed earlier but if a check doesn't do anything for specific profiles, then the rule ought to be removed from the profile itself exactly like how you are doing it for CRY.